build/metadata/github-actions/action.yml

```diff
@@ -1,42 +1,42 @@
-name: 'LaunchDarkly Code References'
-description: 'Find references to feature flags in your code.'
-author: 'LaunchDarkly'
+name: LaunchDarkly Code References
+description: Find references to feature flags in your code.
+author: LaunchDarkly
 branding:
   icon: toggle-right
   color: gray-dark
-inputs:
-  projKey:
-    description: 'Key of the LaunchDarkly project associated with this repository. Found under Account Settings -> Projects in the LaunchDarkly dashboard. Cannot be combined with `projects` block in configuration file.'
-    required: false
-  accessToken:
-    description: 'A token with write access to the LaunchDarkly project.'
+inputs: 
+  accessToken: 
+    description: "A token with write access to the LaunchDarkly project."
     required: true
-  repoName:
-    description: 'The repository name. Defaults to the current GitHub repository.'
+  allowTags: 
+    default: "false"
+    description: "Enable storing references for tags. The tag will be listed as a branch."
+    required: false
+  baseUri: 
+    default: "https://app.launchdarkly.com"
+    description: "The base URL of the LaunchDarkly server for this configuration."
     required: false
-  baseUri:
-    description: 'The base URL of the LaunchDarkly server for this configuration.'
-    default: 'https://app.launchdarkly.com'
+  contextLines: 
+    default: "2"
+    description: "The number of context lines above and below a code reference for the job to send to LaunchDarkly. By default, the flag finder will not send any context lines to LaunchDarkly. If < 0, no source code will be sent to LaunchDarkly. If 0, only the lines containing flag references will be sent. If > 0, will send that number of context lines above and below the flag reference. A maximum of 5 context lines may be provided."
```

Comment:
```suggestion
    description: "The number of context lines above and below a code reference for the job to send to LaunchDarkly. By default, the flag finder will not send any context lines to LaunchDarkly. If < 0, it will send no source code to LaunchDarkly. If 0, it will send only the lines containing flag references. If > 0, it will send that number of context lines above and below the flag reference. You may provide a maximum of 5 context lines."
```